commands/poll.js

```diff
@@ -0,0 +1,151 @@
+const Discord = require('discord.js');
+const emojiRegex = require('emoji-regex');
+
+const symbols = require('../symbols');
+const functions = require('../functions');
+
+function generateTestResults(options) {
+	op = {...options};
+	for (let i = 0; i < Object.keys(op).length; i++) {
+		op[Object.keys(op)[i]] = Math.floor(Math.random() * 100);
+	}
+	return op;
+}
+function generateResults(options) {
+	op = {...options};
+	for (let i = 0; i < Object.keys(op).length; i++) {
+		op[Object.keys(op)[i]] = 0;
+	}
+	return op;
+}
+
+module.exports = {
+	name: 'poll',
+	description: 'Run a poll in a specified channel using reactions',
+	execute(client, message, args) {
+		let poll_options = {};
+		if (args.length > 0) {
+			let guild = client.guilds.cache.get(client.CSIT_ID);
+			let channel = guild.channels.cache.find(ch => ch.name === args.join(' '));
```

Comment:
Are there even spaces in Discord channel names? It shouldn't matter...

Reply:
ikr idk what I was thinking xD